crates/config/src/network.rs

```diff
@@ -102,6 +102,18 @@ pub struct LibP2pConfig {
     pub px_disconnect_timeout: Duration,
 }
 
+impl LibP2pConfig {
```

Comment:
Name is plural, should these return Vec's for future use?  Obviously ok for now and can be dealt this later.